src/metabase/public_settings/premium_features.clj

```diff
@@ -257,9 +261,11 @@
                         :setter     :none
                         :getter     `(default-premium-feature-getter ~(some-> feature name))}
                        options)]
-    `(defsetting ~setting-name
-       ~docstring
-       ~@(mapcat identity options))))
+    `(do
+      (swap! premium-features conj ~feature)
```

Comment:
Have you considered tagging the vars and searching for them instead of introducing this atom?

Reply:
I had not, but I like that idea. I'll try it

Reply:
Actually I'm having some trouble figuring out how to set metadata in the `define-premium-feature` macro. I think I'll keep this approach